src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String output = readFile(fromFileName);
+        writeToFile(output, toFileName);
+    }
+
+    private String readFile(String sourceFile) {
+        File file = new File(sourceFile);
+        int supplyResult = 0;
+        int buyResult = 0;
+        int result = 0;
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+            while (value != null) {
+                if (value.contains(SUPPLY)) {
+                    supplyResult += Integer.parseInt(value.substring(value.indexOf(',') + 1));
+                }
+                if (value.contains(BUY)) {
+                    buyResult += Integer.parseInt(value.substring(value.indexOf(',') + 1));
```

Comment:
[Use split() instead of substring()](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#use-split-instead-of-substring) 

the same for above